ui/push-health/details/DetailsPanel.jsx

```diff
@@ -209,7 +209,7 @@ class DetailsPanel extends React.Component {
                     repoName={currentRepo.name}
                   />
                 </TabPanel>
-                <TabPanel>
+                <TabPanel className="overflow-auto h-100">
```

Comment:
If the list of artifacts is longer than the height of the details panel, it was overflowing.  This is orthogonal to the rest of this PR, but figured I'd fix it while I was here.